Packs/GoogleCloudStorage/Integrations/GoogleCloudStorage/GoogleCloudStorage.yml

```diff
@@ -103,44 +90,33 @@ script:
       - private
       - projectPrivate
       - publicRead
-      required: false
-      secret: false
-    deprecated: false
+    - defaultValue: eu
+      description: The location of the bucket. Object data for objects in the bucket resides in physical storage within this region or multi-region.
+      name: location
+    - description: Whether or not the bucket uses uniform bucket-level access.
+      name: uniform_bucket_level_access
```

Comment:
Add predefined with the options true and false.

Reply:
+1